### ISSUE TITLE
Need help

### ISSUE THREAD

**Issue by mverma16** [Giorgio003/Youtube-TV#30]
Hey guys I am working on a project in which I need to show YouTube videos and playlist. I tried to clone this repository and run demo, but it does not seem to working. Do I need to set API key for running demo also. Please help guys this repository fulfills the requirement of my project and I am unable to use it.

**Comment by m-hatch** [comment]
Yes, you should set the API Key for the demos to work correctly. (I believe the API Key used in the demos currently is restricted or expired.) Be sure to set the key in 'demos/assets/ytv.js' to run the demos from their location - note this is different than the distributable 'src/ytv.js'.
